script.js

```diff
@@ -0,0 +1,105 @@
+// DOM Elements
+const time = document.getElementById('time');
+const greeting = document.getElementById('greeting');
+const name = document.getElementById('name');
+const focus = document.getElementById('focus');
+const SHOW_AM_PM = true;
```

Comment:
I don't understand the meaning of this constant as it is always `true` and in the ternary operator  (line 28) the result of condition-check will be always the same. So, my advice would be to remove it at all and remove ternary operator in line 28.

Reply:
DONE!